tests/unit/test_algebraic_expression.cpp

```diff
@@ -131,23 +131,23 @@ class AlgebraicExpressionTest: public ::testing::Test {
         int labels[11] = {person_label, person_label, person_label, person_label, person_label, person_label,
                         country_label, country_label, country_label, country_label, country_label};
 
-        NodeIterator *it;
+        DataBlockIterator *it;
         Graph_CreateNodes(g, node_count, labels ,&it);
 
         // Assign attributes to nodes.
         char *default_property_name = (char *)"name";
         for(int i = 0; i < person_count; i++) {
-            Node *n = NodeIterator_Next(it);
+            Node *n = (Node*)DataBlockIterator_Next(it);
             SIValue name = SI_StringVal(persons[i]);
             Node_Add_Properties(n, 1, &default_property_name, &name);
```

Comment:
This call crashes for me, and on CircleCI, in a reproducible but strange way when invoked from the `OneIntermidateReturnNode` test (which is not in itself important, but is interesting because the same code is executed by one prior test).

The problem seems to be that the return from `DataBlockIterator_Next` is malloc'ed but not initialized. This can be resolved by using calloc at `data_block.c:43` or memsetting `block->data`, but I don't know if there are better alternatives or anything.